src/tests/artifacts/.shepherd/adds-checked-out-repo/repos-org.json

```diff
@@ -0,0 +1 @@
+[{"name":"test","owner":"NerdWallet"}]
```

Comment:
Nit: could you put these files in a `fixtures` dir in the root of the repo instead of in `src/tests/artifacts`?

Reply:
There will be a trade-off here.  Since the test files require source files, typescript will throw an error TS6059 (i.e. File '...' is not under 'rootDir' ''. 'rootDir' is expected to contain all source files). To overcome this, we can remove 'rootDir' from tsconfig options; however, that would mean setting `bin.shepherd` in package.json equal to './lib/src/cli.js'.

Thoughts?

Reply:
Could we just do a `fs.readFile` + `JSON.parse` instead of a `require`? That should let us avoid any TypeScript problems.

Reply:
Absolutely!  Will do.